pkg/api/goprobe/server/query.go

```diff
@@ -13,7 +13,7 @@ func (server *Server) postQuery(c *gin.Context) {
 	api.RunQuery(
 		fmt.Sprintf("goProbe/%s", version.Short()),
 		"local DB",
-		engine.NewQueryRunner(server.dbPath),
+		engine.NewQueryRunnerWithLiveData(server.dbPath, server.captureManager),
```

Comment:
Maybe I misunderstood: does this query both the DB AND the live data or just the live data?

Reply:
Never mind. I understood